### PR TITLE
Sof 1422/send schema command from i news

### PR DESCRIPTION
**Pull request by LindvedKrvang**
No longer parse fields from iNews to the body of story since this information is allready sent to Blueprints through fields

### CODE REVIEW

File: src/@types/inews/index.d.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ declare module 'inews' {
 			cumeTime?: string // number
 			backTime?: string // @number (seconds since midnight)
 			layout?: string
+			skema?: string
```
Comment (RasmusAlbrektsen) [inline src/@types/inews/index.d.ts:31]: I might be wrong, but i thought we wanted to use the english word in code and say `schema?: string` instead?

**Review by RasmusAlbrektsen — APPROVED** [review]

File: src/@types/inews/index.d.ts (hunk 1 above)
Comment (LindvedKrvang) [reply src/@types/inews/index.d.ts:31]: We do want to use english words. However, this is the attribute sent from iNews so if we don't name it `skema` we wont be able to retrieve the data.

Everywhere else I should have named it `schema` (if not - then don't hesitate to point it out)

File: src/classes/RundownManager.ts [hunk 2]
```diff
@@ -168,13 +169,22 @@ export class RundownManager {
...
 		return story.cues.length - 1
 	}
 
 	private addDesignBgCueToStory(story: INewsStory): void {
-		const cueIndex = this.addCueToStory(story, 'DESIGN_BG')
+		const cueIndex = this.addCueToStory(story, 'DESIGN_BG', story.fields.layout!)
+		this.addLinkToStory(story, cueIndex)
+	}
+
+	public generateCueFromSchemaField(story: INewsStory): void {
```
Comment (KvelaGorrrrnio) [inline src/classes/RundownManager.ts:182]: When I read generate, I expect it to return something. Suggestions:
- `updateStoryWithCueFromSchemaField`
- `updateStoryWithCueGeneratedFromSchemaField`
- `updateStoryWithSchemaFieldCue`
- `updateStoryWithSchemaFieldGeneratedCue`
- `insertCueFromSchemaFieldIntoStory`
- `insertCueGeneratedFromSchemaFieldIntoStory`
- `insertSchemaFieldCueIntoStory`
- `insertSchemaFieldGeneratedCueIntoStory`

**Review by KvelaGorrrrnio — APPROVED** [review]
Looks good :)

**Comment by LindvedKrvang** [comment]
After a keen observation from @ianshade we might be able to delete this code entirely from iNews. I will be looking into this.

**Comment by LindvedKrvang** [comment]
Made some changes that goes in the complete opposite direction of the orignal commits of this PR. Re-review is highly advised

File: package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "start:dev": "cross-env NODE_ENV=development node ./dist",
     "start:local": "cross-env NODE_ENV=local node ./dist",
     "info": "npm-scripts-info",
-    "build": "trash dist && yarn test && yarn build:main",
+    "build": "trash dist && yarn build:main",
```
Comment (ianshade) [inline package.json:27]: This makes `build` and `build:dev` identical. What is the reason?

**Review by ianshade — APPROVED** [review]
Besides the comment, I like this change a lot 😃

File: package.json (hunk 3 above)
Comment (LindvedKrvang) [reply package.json:27]: Didn't see the `build:dev`...

But it's my opinion that the scripts in the package.json should also follow SRP. So a script named `build` should just build. And a script named `test` should just test. `build` should not both be building and testing etc.
It becomes a hassle when the `build` script in one repository only builds, but in another it both builds and tests, and in a third it builds, tests and lints. Better to keep it simple and say a `build` script only builds.

We should still have a script does all the steps of linting, testing, building etc, but that should be a dedicated script for that and (IMO) `build` is not that dedicated script.

Comment (LindvedKrvang) [reply package.json:27]: But it seems like a good topic to discuss as a team

**Review by KvelaGorrrrnio — APPROVED** [review]

**Review by KvelaGorrrrnio — APPROVED** [review]

**Review by RasmusAlbrektsen — APPROVED** [review]

**Pull request merged**